data/categories.txt

```diff
@@ -3629,7 +3635,7 @@ ru:^Мусорный контейнер|мусорка
 ar:^قمامة|مهملات|سلة مهملات|صندوق قمامة
 cs:^Odpadky|koš|popelnice|uložiště odpadu
 da:^Bortskaffelse af affald|skraldespand|skrald
-nl:^Uitschot|afval|prullenbak|vuilnis
+nl:^Afval container|afval|prullenbak|vuilnis
```

Comment:
I think this should be 'afval container'

Reply:
Thank you!